build.gradle

```diff
@@ -19,6 +19,18 @@ plugins {
   id 'idea' // Support for IntelliJ IDEA projects
 }
 apply plugin: 'kotlin'
+apply plugin: 'jacoco'
+jacoco {
+    toolVersion = "0.7.6.201602180812"
+    reportsDir = file("build/customJacocoReportDir")
+}
```

Comment:
I don't think we need to specify the tool version and the reportsDir.
In general I'd stick with the default values, unless we need specific custom configurations.